projects/Odroid_C2/linux/linux.aarch64.conf

```diff
@@ -2577,13 +2656,13 @@ CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 #
 # Mali GPU OpenGL device driver
 #
-CONFIG_MALI400=y
+CONFIG_MALI400=m
 # CONFIG_MALI400_DEBUG is not set
 # CONFIG_MALI400_PROFILING is not set
 # CONFIG_MALI400_UMP is not set
```

Comment:
We also probably should have UMP set here as this will enable ways to get a proper playback with zerocopy